nomad-base/bin/example.rs

```diff
@@ -1,37 +1,47 @@
 use color_eyre::Result;
 
-use nomad_base::{NomadAgent, Settings};
-
-/// An example main function for any agent that implemented Default
-async fn _example_main<OA>(settings: Settings) -> Result<()>
-where
-    OA: NomadAgent<Settings = Settings> + Sized + 'static,
-{
-    // Instantiate an agent
-    let oa = OA::from_settings(settings).await?;
-    oa.as_ref()
-        .settings
-        .tracing
-        .start_tracing(oa.metrics().span_duration())?;
-
-    // Use the agent to run a number of replicas
-    oa.run_all().await?
-}
-
-/// Read settings from the config file and set up reporting and logging based
-/// on the settings
-#[allow(dead_code)]
-fn setup() -> Result<Settings> {
-    color_eyre::install()?;
-
-    let settings = Settings::new()?;
-
-    Ok(settings)
-}
-
-#[allow(dead_code)]
+// use nomad_base::{decl_settings, NomadAgent};
```

Comment:
maybe just delete the file at this point. no need for it

Reply:
https://github.com/nomad-xyz/rust/issues/38